pango/Gir.toml

```diff
@@ -270,6 +270,12 @@ concurrency = "send+sync"
     [[object.function]]
     name = "merge_static"
     ignore = true
+    [[object.function]]
```

Comment:
there are examples in gtk4-rs on how to set self as const without manual implementation btw; you just use the c param name

Reply:
It's not about that, but the string parameter needs to have a static lifetime